### PR TITLE
Accessibility fixes for Naming Styles

### PR DESCRIPTION
**Pull request by dpoeschl**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems?_a=edit&id=391338&triage=true & https://devdiv.visualstudio.com/DevDiv/_workitems?_a=edit&id=391337&triage=true

### CODE REVIEW

**Comment by dpoeschl** [comment]
Tagging for review @dotnet/roslyn-ide

File: src/VisualStudio/Core/Def/ServicesVSResources.resx [hunk 1]
```diff
@@ -876,4 +876,25 @@ Additional information: {1}</value>
...
+    <value>Edit item.</value>
+  </data>
+  <data name="Remove_item" xml:space="preserve">
+    <value>Remove item.</value>
+  </data>
+  <data name="Add_a_naming_rule" xml:space="preserve">
+    <value>Add a naming rule.</value>
+  </data>
+  <data name="Remove_naming_rule" xml:space="preserve">
+    <value>Remove naming rule.</value>
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx]: are all the trailing periods intentional/desirable here?

File: src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageControl.xaml.cs [hunk 2]
```diff
@@ -24,6 +24,11 @@ internal partial class NamingStyleOptionPageControl : AbstractOptionPageControl
         public static string RequiredStyleHeader => ServicesVSResources.Required_Style;
         public static string SeverityHeader => ServicesVSResources.Severity;
         public static string ExplanatoryText => ServicesVSResources.For_a_given_symbol_only_the_topmost_rule_with_a_matching_Specification_will_be_applied_Violation_of_that_rules_Required_Style_will_be_reported_at_the_chosen_Severity_level;
+        public static string AddRuleAutomationText => ServicesVSResources.Add_a_naming_rule;
+        public static string RemoveAutomationText => ServicesVSResources.Remove_naming_rule;
+        public static string SymbolSpecification => ServicesVSResources.Symbol_Specification;
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageControl.xaml.cs]: are the last tree also AutomationText?

File: src/VisualStudio/Core/Def/ServicesVSResources.resx (hunk 1 above)
Comment (dpoeschl) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx]: I completely don't care. It doesn't change how anything gets read.

File: src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageControl.xaml.cs (hunk 2 above)
Comment (dpoeschl) [inline src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageControl.xaml.cs]: I'll fix up the naming

**Pull request merged**